### PR TITLE
Use api show product

### PR DESCRIPTION
**Pull request by at-khadinh**
## Description
Use api to show product detail.
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: public/js/public/product.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+function loadPage () {
```
Comment (at-mailuong) [inline public/js/public/product.js]: dư space

File: public/js/public/product.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+            let imagePri = '<img id="example" src="' + imagePath + response.result.images[0].img_url + '" alt=" " class="img-responsive">';
+            $('.agileinfo_single .col-md-4 .agileinfo_single_left').append(imagePri);
+
+            response.result.images.forEach(image => {
+                imageSub = '<li class="col-md-4 sub-images-item"><div><img id="example" src="' + imagePath + image.img_url + '" alt=" " class="img-responsive"></div></li>';
+                $('.sub-images .sub-images-list').append(imageSub);
+            });
+
+            for(i = 1; i <= 5; i++){
```
Comment (at-mailuong) [inline public/js/public/product.js]: space nha , cái if ở dưới nữa

**Comment by at-mailuong** [comment]
LGTM

File: public/js/public/product.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+            let imagePri = '<img id="example" src="' + imagePath + response.result.images[0].img_url + '" alt=" " class="img-responsive">';
+            $('.agileinfo_single .col-md-4 .agileinfo_single_left').append(imagePri);
+
+            response.result.images.forEach(image => {
+                imageSub = '<li class="col-md-4 sub-images-item"><div><img id="example" src="' + imagePath + image.img_url + '" alt=" " class="img-responsive"></div></li>';
+                $('.sub-images .sub-images-list').append(imageSub);
+            });
+
+            for (i = 1; i <= 5; i++) {
```
Comment (at-huenguyen) [inline public/js/public/product.js]: `5` ở đâu ra vại em?

File: resources/views/public/pages/detail.blade.php [hunk 4]
```diff
@@ -19,54 +19,39 @@
...
             <div class="snipcart-details agileinfo_single_right_details">
               <form action="#" method="post">
                 <fieldset>
                   <input type="hidden" name="cmd" value="_cart">
                   <input type="hidden" name="add" value="1">
                   <input type="hidden" name="business" value=" ">
-                  <input type="hidden" name="item_name" value="pulao basmati rice">
-                  <input type="hidden" name="amount" value="21.00">
-                  <input type="hidden" name="discount_amount" value="1.00">
+                  <input type="hidden" name="item_name" value=" ">
```
Comment (at-huenguyen) [inline resources/views/public/pages/detail.blade.php]: em có thể để `""` thay cho `" "`

**Review by ghost — APPROVED** [review]

**Pull request merged**